### PR TITLE
Split ENTRYPOINT script in 2

### PR DESCRIPTION
**Pull request by mscherer**
ENTRYPOINT is made to be used when the container is running one specific executable (think bash, or python) and is hard to override. However, when using the container, it is sometime needed to execute specific subcommand, such as augur db upgrade-db-version

**Signed commits**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: docker/backend/init.sh
Comment (sgoggins) [inline docker/backend/init.sh:1]: @ABrain7710 : do we support this way of loading anymore, or does it need to be depricated?

**Review by sgoggins — COMMENTED** [review]
@mscherer : I have one question for one of our other maintainers I asked in context. Waiting for an answer.

**Review by ABrain7710 — APPROVED** [review]

**Pull request merged**
